src/containers/CreateDoc/index.tsx

```diff
@@ -16,11 +19,30 @@ const CreateDoc: React.FC = () => {
     return (
         <React.Fragment>
             <h1>CreateDoc Page</h1>
-            <Dropdown name="documentType" title="Document type" options={documentTypes} />
+            <Dropdown 
+                action={props.setDocType} 
+                name="documentType" 
+                options={documentTypes} 
+                title="Document type" 
+            />
             <button onClick={openPdf}>See preview</button>
             <button onClick={downloadPdf}>Download PDF</button>
+            <p>Selected doc type: {props.docType}</p>
         </React.Fragment>
     );
 }
 
-export default CreateDoc;
\ No newline at end of file
+const mapDispatch = {
+    setDocType: (docType: string) => setDocType(docType)
```

Comment:
can you explain this ?  this is a const without a type that has a key, with a function that should return the result of `setDocType`, but isn't that the same name of the key ?

Reply:
The first setDocType it's the name that we're giving to the prop, and the second one refers to the action. I will rename the second to setDocTypeAction.

Reply:
Should that const have a type ?

Reply:
yes, otherwise TS complains:

Parameter 'docType' implicitly has an 'any' type.